pkg/protocol/sybilprotection/sybilprotectionv1/performance/testsuite_test.go

```diff
@@ -260,14 +264,38 @@ func (t *TestSuite) calculatePoolReward(epoch iotago.EpochIndex, totalValidators
 	params := t.api.ProtocolParameters()
 	targetReward, err := params.RewardsParameters().TargetReward(epoch, t.api)
 	require.NoError(t.T, err)
-
 	poolCoefficient := t.calculatePoolCoefficient(poolStake, totalStake, validatorStake, totalValidatorsStake)
 	scaledPoolReward := poolCoefficient * uint64(targetReward) * performanceFactor
 	poolRewardNoFixedCost := scaledPoolReward / uint64(params.ValidationBlocksPerSlot()) >> (params.RewardsParameters().PoolCoefficientExponent + 1)
 
 	return poolRewardNoFixedCost
 }
 
+func (t *TestSuite) calculatePoolRewardFloat(epoch iotago.EpochIndex, totalValidatorsStake, totalStake, poolStake, validatorStake iotago.BaseToken, performanceFactor uint64) float64 {
+	params := t.api.ProtocolParameters()
+	totalValidatorsStakeFloat := float64(totalValidatorsStake)
+	totalStakeFloat := float64(totalStake)
+	poolStakeFloat := float64(poolStake)
+	validatorStakeFloat := float64(validatorStake)
+	performanceFactorFloat := float64(performanceFactor) / float64(params.ValidationBlocksPerSlot())
+	// theta := float64(params.RewardsParameters().ManaShareCoefficient)
+	// S := float64(params.TokenSupply())
+	// gamma := float64(params.ManaParameters().GenerationRate) * math.Pow(2.0, -float64(params.ManaParameters().GenerationRateExponent))
+	d := math.Pow(2.0, float64(params.SlotsPerEpochExponent()))
+	betaDelta := d * float64(params.SlotDurationInSeconds()) / 3.0 / (365 * 24 * 60 * 60)
+
+	targetRewardFloat := float64(t.api.ComputedFinalReward())
+
+	// targetRewardFloat := theta * S * gamma * d
```

Comment:
We should get this part to work. As we discussed, this probably has something to do with different protocol parameters being used in different places, but otherwise there is a mistake somewhere in either calculation, so I think we need to figure this out rather than just leave it out.